lib/galaxy/config/__init__.py

```diff
@@ -615,11 +615,7 @@ def _process_config(self, kwargs):
             log.warning("preserve_python_environment set to unknown value [%s], defaulting to legacy_only")
             self.preserve_python_environment = "legacy_only"
         self.nodejs_path = kwargs.get("nodejs_path")
-        # Older default container cache path, I don't think anyone is using it anymore and it wasn't documented - we
-        # should probably drop the backward compatiblity to save the path check.
-        self.container_image_cache_path = self._in_data_dir(kwargs.get("container_image_cache_path", "container_images"))
-        if not os.path.exists(self.container_image_cache_path):
-            self.container_image_cache_path = self._in_root_dir(kwargs.get("container_image_cache_path", self._in_data_dir("container_cache")))
+        self.container_image_cache_path = self._in_data_dir(kwargs.get("container_image_cache_path", "container_cache"))
```

Comment:
Should `container_image_cache_path` go to galaxy.yml.sample? @mvdbeek @nsoranzo

Reply:
No, this is a legacy parameter and is overwritten by container_resolvers_conf.xml which gives you much more flexibility. See https://training.galaxyproject.org/training-material/topics/admin/tutorials/singularity/tutorial.html for details.

Reply:
Yes, I think it should be added to `lib/galaxy/webapps/galaxy/config_schema.yml`

Reply:
After pulling this change our Galaxy failed to load because it wanted to write to `/mnt/volume/shared/galaxy/server/database/container_cache` that was never used before.

Reply:
@martenson Was your Galaxy using `/mnt/volume/shared/galaxy/server/database/container_images` ?

Reply:
> No, this is a legacy parameter and is overwritten by container_resolvers_conf.xml which gives you much more flexibility.

I don't agree, it's still functional AFAIU so it should be documented as legacy "please use `container_resolvers_conf.xml` (or removed).

Reply:
@nsoranzo no, we are not using `/mnt/volume/shared/galaxy/server/database` at all.

Reply:
@martenson That can only come from your `data_dir` directory, I think. I've looked at this code a few times and it seems the standard way we treat similar config options.